adc/ad7779/imxrt/spi.c

```diff
@@ -0,0 +1,58 @@
+#include <unistd.h>
+#include <sys/msg.h>
+
+#include <phoenix/arch/imxrt.h>
+#include <imxrt-multi.h>
+
+#include "../ad7779.h"
+
+oid_t spi_driver = { 0 };
+
+int spi_exchange(uint8_t *buff, uint8_t len)
+{
+	msg_t msg = { 0 };
+	multi_i_t *imsg = (multi_i_t *)msg.i.raw;
+	multi_o_t *omsg = (multi_o_t *)msg.o.raw;
+
+	msg.type = mtDevCtl;
+	msg.i.data = buff;
+	msg.i.size = len;
+	msg.o.data = buff;
```

Comment:
shouldn't we just put the buffer once (in `msg.o`?), @agkaminski how it should be done?

Reply:
I put some new lines

Reply:
@agkaminski ?